py/selenium/webdriver/common/bidi/network.py

```diff
@@ -0,0 +1,125 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from .session import session_subscribe
+from .session import session_unsubscribe
+
+
+class Network:
+    EVENTS = {
+        'before_request': 'network.beforeRequestSent',
+        'response_started': 'network.responseStarted',
+        'response_completed': 'network.responseCompleted',
+        'auth_required': 'network.authRequired',
+        'fetch_error': 'network.fetchError'
+    }
+
+    PHASES = {
+        'before_request': 'beforeRequestSent',
+        'response_started': 'responseStarted',
+        'auth_required': 'authRequired'
+    }
+
+    def __init__(self, conn):
+        self.conn = conn
+        self.callbacks = {}
+
+    def continue_response(self, request_id, status_code, headers=None, body=None):
```

Comment:
All the methods except for the ones mentioned in https://github.com/SeleniumHQ/selenium/issues/13993 should be private.